benches/hash.rs

```diff
@@ -0,0 +1,56 @@
+
+
+use criterion::{black_box, criterion_group, criterion_main, BatchSize, Criterion};
+use miden_crypto::{hash::{Hasher, Digest}, HashFn, Felt, ElementHasher};
+use rand_utils::rand_value;
+
+fn rpo256_2to1(c: &mut Criterion) {
+    let v: [Digest; 2] = [Hasher::hash(&[1u8]), Hasher::hash(&[2u8])];
```

Comment:
minor nit: when adding types to constants we use the following convention `value_type`. For example, `2_u8` rather than `2u8`.

Reply:
Got it, and noted  for the future.